Cargo.toml

```diff
@@ -58,238 +58,301 @@ iroha_trigger_derive = { version = "=2.0.0-pre-rc.19", path = "smart_contract/tr
+rust.trivial_casts = "deny"
+rust.trivial_numeric_casts = "deny"
+rust.unconditional_recursion = "deny"
+rust.unsafe_code = "deny"
+rust.unused = "deny"
+rust.unused_import_braces = "deny"
+rust.variant_size_differences = "deny"
+rust.unused_tuple_struct_fields = "deny"
+rust.explicit_outlives_requirements = "deny"
+rust.non_ascii_idents = "deny"
+rust.elided_lifetimes_in_paths = "allow"
+rust.unknown_lints = "warn"
+rust.single_use_lifetimes = "warn"
+rust.unused_lifetimes = "warn"
 
 # lower the priority to allow overriding later
 clippy.pedantic = { level = "deny", priority = -1 }
+
+# lower the priority to allow overriding later
 clippy.all = { level = "deny", priority = -1 }
+
+# lower the priority to allow overriding later
 clippy.dbg_macro = "deny"
 
-# clippy.nursery = "deny"
+# lower the priority to allow overriding later
 clippy.debug_assert_with_mut_call = "deny"
+
+# lower the priority to allow overriding later
 clippy.derive_partial_eq_without_eq = "deny"
+
+# lower the priority to allow overriding later
 clippy.empty_line_after_outer_attr = "deny"
+
+# lower the priority to allow overriding later
 clippy.fallible_impl_from = "deny"
+
+# lower the priority to allow overriding later
 clippy.future_not_send = "deny"
+
+# lower the priority to allow overriding later
 clippy.iter_with_drain = "deny"
+
+# lower the priority to allow overriding later
 clippy.mutex_integer = "deny"
+
+# lower the priority to allow overriding later
 clippy.needless_collect = "deny"
+
+# lower the priority to allow overriding later
 clippy.path_buf_push_overwrite = "deny"
+
+# lower the priority to allow overriding later
 clippy.suboptimal_flops = "deny"
+
+# lower the priority to allow overriding later
 clippy.trailing_empty_array = "deny"
+
+# lower the priority to allow overriding later
 clippy.transmute_undefined_repr = "deny"
+
+# lower the priority to allow overriding later
 clippy.trivial_regex = "deny"
+
+# lower the priority to allow overriding later
 clippy.unused_peekable = "deny"
-clippy.unused_rounding = "deny"
 
-rust.future_incompatible = "deny"
-rust.missing_copy_implementations = "deny"
-rust.missing_docs = "deny"
-rust.nonstandard_style = "deny"
-rust.private_doc_tests = "deny"
-rust.rust_2018_idioms = "deny"
-rust.trivial_casts = "deny"
-rust.trivial_numeric_casts = "deny"
-rust.unconditional_recursion = "deny"
-rust.unsafe_code = "deny"
-rust.unused = "deny"
-rust.unused_import_braces = "deny"
-rust.variant_size_differences = "deny"
-rust.unused_tuple_struct_fields = "deny"
-rust.explicit_outlives_requirements = "deny"
-rust.non_ascii_idents = "deny"
-# TODO: reenable
-# rust.unreachable_pub = "deny"
-# rust.unsafe_op_in_unsafe_fn = "deny"
+# lower the priority to allow overriding later
+clippy.unused_rounding = "deny"
 
-# These are up to personal taste. We don't want these to be enabled ever.
+# lower the priority to allow overriding later
 clippy.string_add = "allow"
+
+# lower the priority to allow overriding later
```

Comment:
Why did you leave this comment for every lint? I think it was on a right place before

Reply:
Maybe some accident automation?

Reply:
interesting, no idea what happened